src/header/header.scss

```diff
@@ -337,7 +301,7 @@
     @include iui-header-button-active;
 
     // Removing button hovers and backgrounds and adding last child hover
-    .iui-button {
+    .iui-header-breadcrumbs-button {
       &.iui-borderless {
```

Comment:
dont think we use now iui-borderless anywhere.. all this might be gone now (unless now smsth is broken..)

Reply:
All mentions of `.iui-borderless` have been removed from the header CSS.